packages/schematics/angular/application/files/__dot__angular-cli.json

```diff
@@ -51,6 +51,9 @@
     }
   },
   "defaults": {
+    "build": {
+      "ignored": ["**/*~", "**/.#*", "**/.*.sw[pon]"]
```

Comment:
Actually, nevermind my last comment. We don't have an option in the configuration for `ignored`. Where did you get that field from?